test/NuGet.Core.Tests/NuGet.Packaging.Test/Core/comparers/PackageDependencyInfoComparerTests.cs

```diff
@@ -120,8 +119,8 @@ public void GetHashCode_WhenArgumentIsValid_ReturnsHashCode()
         {
             var combiner = new HashCodeCombiner();
 
-            combiner.AddObject(PackageIdentityComparer.Default.GetHashCode(DependencyInfoA));
-            combiner.AddObject(DependencyInfoA.Dependencies.First());
+            combiner.AddObject(DependencyInfoA, PackageIdentityComparer.Default);
+            combiner.AddUnorderedSequence(DependencyInfoA.Dependencies);
```

Comment:
The test had to change here because `AddUnorderedSequence` includes the count of items in the hash (which is required to prevent errors when using XOR), and the previous code just added each item. So the test could previously just add the single dependency and have it match, but now it needs to call `AddUnorderedSequence` so that the count can be included too.